.flatpak-appdata.xml

```diff
@@ -30,6 +30,9 @@
     </ul>
     <p>You can also bring new features with Podman Desktop plug-ins or Docker Desktop extensions.</p>
   </description>
+  <categories>
+    <category>Development</category>
```

Comment:
What about the metadata in .flatpak.desktop ?

Reply:
@jeffmaury it's unclear to me whether or not the flatpak-appdata.xml needs this category, I saw some documents saying it does, but the official appdata spec does not mention it. I can ping a flathub admin and see what they say.

Reply:
What I was questioning is that it seems to me that the 2 files seem to be out of sync because .flatfak.desktop also has a Categories attribute